test/arbitrage/BancorArbitrage.ts

```diff
@@ -220,14 +220,10 @@ describe('BancorArbitrage', () => {
 
         describe('distribution and burn', () => {
             // get all exchange ids (omit their names)
-            const exchangeIds = Object.values(ExchangeId).filter((key) => !isNaN(parseInt(key as string)));
```

Comment:
this prevents double keys to be in the list of ids, you sure it's not needed?

Reply:
Yes, exchangeIds for the distribution and burn tests are not needed.